lesson07/assignment/src/views/accordion.vue

```diff
@@ -0,0 +1,151 @@
+                                <div class="modal-dialog modal-lg">
+                                    <div class="modal-content">
+                                     {{poster.modal_line_1}}<br>
+                                     {{poster.modal_line_2}}<br>
+                                     {{poster.modal_line_3}}<br>
+                                     {{poster.modal_line_4}}<br>
+                                     {{poster.modal_line_5}}<br>
+                                     {{poster.modal_line_6}}<br>
+                                     {{poster.modal_line_7}}<br>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="display2 comments" alt="To add a comment refresh the page">
+                <p v-if="show" ><span>{{nickname}}: </span> {{comment_text}}</p>
+                <div v-else="show"class="comment_form">
+                    <input type="text" v-model="nickname"  placeholder='Type your nickname'>
+                    <textarea type="text" v-model="comment_text"  placeholder='Type your comment' > </textarea>
+                    <button v-on:click="show = !show"> Add test comment</button>
+                </div>
+            </div>
+        </section>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+
+#accordion{
+    display: inline-block;
+    display: flex;
+    justify-content:space-around;
+    padding: 2% 0;
+    align-items:    baseline; 
+    img{
+        width: 400px;
+        height: 550px;
+        object-fit: cover;  
+    }
+    .card-body{
+        padding: 0;
+    }
+    // Styling modal button
+    h3{
+        padding: 2% 0;
+        font-weight: bold;
+        margin: 0;
+        cursor: pointer;
+        text-align: center;
+        font-size: 2.3em;
+        transition: 0.5s;
+        &:hover{
+            background:lightblue;
+            color: white;
+        }
+    }
+}
+.modal-content{
+    font-size: 2em;
+    padding: 5% 10%;
+}
+.comments{
+
+    p{
+        span{
+            font-weight: bold;
+        }
+    }
+    .comment_form{
+        display: flex;
+        flex-direction: column;
+        width: 20%;
+        margin: 0 auto;
+        input, textarea{
+            text-align: center;
+        }
+    }
+}
+</style>
+
+<script>
+    export default{
+        name: 'accordion',
+        data() {
+            return{
+                show: false,
+                posters: [
+                {
+                img_link : require('../assets/anime-1.jpg'),
+                div_id : 'one',
+                movie_name: 'Your Name',
+                div_class : 'bd-example-modal-lg',
+
+                modal_line_1 : 'A teenage boy and girl embark on a quest to meet each other for the first time after they magically swap bodies.',
```

Comment:
Alternatively, you can make an array of `modal_line:` and loop over it with another `v-for` to output your lines.